install/resources/upgrade52.php

```diff
@@ -31,13 +31,15 @@ function upgrade51_dbchanges()
 
 	switch($db->type)
 	{
-		// Add new setting for new usergroup permission if group members can hide online status
-		case "pgsql":
-			$db->add_column("usergroups", "canbeinvisible", "smallint NOT NULL default '1' AFTER canusercp");
-			break;
-
-		default:
-			$db->add_column("usergroups", "canbeinvisible", "tinyint(1) NOT NULL default '1' AFTER canusercp");
-			break;
-	}
+        // Add new settings for (a) new usergroup permission if group members can hide online status & (b) attachments force download
+        case "pgsql":
```

Comment:
Nitpick: indentation should be tabs, not spaces here - that's why the diff looks so odd.